ebr_connector/prepacked_queries/test_results.py

```diff
@@ -0,0 +1,94 @@
+"""
+A collection of queries that provide failing test results
+"""
+
+from elasticsearch_dsl import Q
+
+from ebr_connector.prepacked_queries.query import make_query
+from ebr_connector.prepacked_queries.flaky_jobs import get_dict_value
+
+
+MAXIMUM_NUM_RECORDS = 10000
+
+JOB_FIELDS = {
+    "includes": [
+        "br_build_date_time",
+        "br_job_name",
+        "br_job_info",
+        "collector",
+        "br_product",
+        "br_build_id_key",
+        "br_platform",
+        "br_product_version_key",
+        "br_tests_object.br_tests_failed_object.br_test",
+        "br_tests_object.br_suites_object.br_package",
```

Comment:
Do you need to get this suites object, I don't think it's used yet. Please make sure you only get the fields you use.